src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -133,7 +133,7 @@ private void OnCurrentWorkspaceChanged(IWorkspaceModel workspace)
         {
             NotificationTracker.Remove(CurrentWorkspace.Guid);
             CurrentWorkspace = workspace as WorkspaceModel;
-            if (PythonDependencies.ContainsIronPythonDependencies())
+            if (PythonDependencies.ContainsIronPythonDependencies() && Configuration.DebugModes.IsEnabled("Python3DebugMode"))
```

Comment:
Adding debug mode check

Reply:
We should change the order of these conditions. There is no need to check for dependencies if the debug mode is disabled.

Reply:
Sure